public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -907,30 +908,39 @@ public SurveyQuestionResult apply(AchillesResult o) {
         return ResponseEntity.ok(TO_CLIENT_ACHILLES_RESULT.apply(result));
     }
 
-    public TreeSet<Float> makeBins(Float min,Float max) {
-        TreeSet<Float> bins = new TreeSet<>();
+    public ArrayList<Float> makeBins(Float min,Float max) {
+        TreeSet<String> bins = new TreeSet<>();
         float binWidth = (max-min)/11;
 
-        if (min >= 10 && max >= 10) {
-            double tempBinWidth = Math.ceil(binWidth/10)*10;
-            binWidth = (float)tempBinWidth;
-        } else if ((max-min) <= 1 && (max-min) >= 0.1) {
-            double tempBinWidth = Math.ceil(binWidth/0.1)*0.1;
-            binWidth = (float)tempBinWidth;
+        if (binWidth >= 5) {
```

Comment:
This is complex to explain. 
If we have min & max of 12 and 392 we are rounding them to multiples of 10 for better readability.
That makes them 10 and 400.
And when we calculate bin_width (400-10/11) => 35.4 we do not want buckets to be 10-45.4, 45.4-80, ..
We want them to be 10-45, 45-80,..
so rounding bin_width to multiples of 5.
Next rounding bin_width to multiples of 0.1 if it is between 0.1 - 1.
Not dealing with rest of them right now, so just leaving them as they are.